scripts/README.md

```diff
@@ -1,6 +1,6 @@
 # Scripts
```

Comment:
there's a small typo in this file on wandb as "wadnb", sorry not strictly from this PR

Reply:
fixed!